google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java

```diff
@@ -913,8 +898,10 @@ public void awaitReplication(String tableId) {
     com.google.bigtable.admin.v2.TableName tableName =
         com.google.bigtable.admin.v2.TableName.of(projectId, instanceId, tableId);
 
+    CheckConsistencyParams params = new CheckConsistencyParams(tableName, CheckConsistencyParams.CheckConsistencyMode.STANDARD);
```

Comment:
From the enduser perspective, they are calling awaitConsistency, why are the params CheckConsistencyParams?

Reply:
Fair - removed "Check"